src/sizzle.js

```diff
@@ -8,2275 +8,2393 @@
+					( nodeType !== 1 || context.nodeName.toLowerCase() !== "object" ) ) {
 
-					push.apply( results, context.getElementsByClassName( m ) );
-					return results;
-				}
-			}
+					newSelector = selector;
+					newContext = context;
 
-			// Take advantage of querySelectorAll
-			if ( support.qsa &&
-				!nonnativeSelectorCache[ selector + " " ] &&
-				(!rbuggyQSA || !rbuggyQSA.test( selector )) &&
-
-				// Support: IE 8 only
-				// Exclude object elements
-				(nodeType !== 1 || context.nodeName.toLowerCase() !== "object") ) {
-
-				newSelector = selector;
-				newContext = context;
-
-				// qSA considers elements outside a scoping root when evaluating child or
-				// descendant combinators, which is not what we want.
-				// In such cases, we work around the behavior by prefixing every selector in the
-				// list with an ID selector referencing the scope context.
-				// Thanks to Andrew Dupont for this technique.
-				if ( nodeType === 1 && rdescend.test( selector ) ) {
-
-					// Capture the context ID, setting it first if necessary
-					if ( (nid = context.getAttribute( "id" )) ) {
-						nid = nid.replace( rcssescape, fcssescape );
-					} else {
-						context.setAttribute( "id", (nid = expando) );
-					}
+					// qSA considers elements outside a scoping root when evaluating child or
+					// descendant combinators, which is not what we want.
+					// In such cases, we work around the behavior by prefixing every selector in the
+					// list with an ID selector referencing the scope context.
+					// Thanks to Andrew Dupont for this technique.
+					if ( nodeType === 1 && rdescend.test( selector ) ) {
 
-					// Prefix every selector in the list
-					groups = tokenize( selector );
-					i = groups.length;
-					while ( i-- ) {
-						groups[i] = "#" + nid + " " + toSelector( groups[i] );
-					}
-					newSelector = groups.join( "," );
+						// Capture the context ID, setting it first if necessary
+						if ( ( nid = context.getAttribute( "id" ) ) ) {
+							nid = nid.replace( rcssescape, fcssescape );
+						} else {
+							context.setAttribute( "id", ( nid = expando ) );
+						}
 
-					// Expand context for sibling selectors
-					newContext = rsibling.test( selector ) && testContext( context.parentNode ) ||
-						context;
-				}
+						// Prefix every selector in the list
+						groups = tokenize( selector );
+						i = groups.length;
+						while ( i-- ) {
+							groups[ i ] = "#" + nid + " " + toSelector( groups[ i ] );
+						}
+						newSelector = groups.join( "," );
 
-				try {
-					push.apply( results,
-						newContext.querySelectorAll( newSelector )
-					);
-					return results;
-				} catch ( qsaError ) {
-					nonnativeSelectorCache( selector, true );
-				} finally {
-					if ( nid === expando ) {
-						context.removeAttribute( "id" );
+						// Expand context for sibling selectors
+						newContext = rsibling.test( selector ) &&
+							testContext( context.parentNode ) || context;
```

Comment:
If you're putting this in a new line, I'd put the `|| context` part in a new line as well to not confuse evaluation precedence. But please re-check if it doesn't fit after de-indenting the file.

Reply:
I'll review after reindent the whole file.

Reply:
Hi @mgol finally I re-indented the `sizzle.js` file. About `|| context`, according with the `eslint` configuration the operator `||` have to be placed in the parent line. Is ok for you?

Reply:
Yes, that sounds OK. I added such a comment in the new review so that it doesn't get lost in outdated topics, I'm resolving this one.